templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  <h1>Delete Car</h1>
+
+  <p>Are you sure you want to delete the car: {{ car }}?</p>
+
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Delete" class="btn btn-danger">
+    <a href="{% url 'taxi:car-list' %}" class="btn btn-primary">Cancel</a>
+  </form>
+
+{% endblock %}
```

Comment:
no blank line

Reply:
fix this everywhere

Reply:
![image](https://github.com/mate-academy/py-taxi-service-forms/assets/84961728/43b19fe5-854e-489b-9495-64fcfbde7462)

I have blank lines everywhere in my PyCharm project, but I can edit files right here, if you want to